### PR TITLE
Update Protocol Locator in various Reducers

### LINKED ISSUE #619: Funding current fails
It looks like protocol routing for a `NewLedgerFunding` action is not working correctly and causes funding to fail with the following error:
```Uncaught TypeError: Cannot read property 'type' of undefined
    at Object.isFundingState (states.ts:116)
    at isFundingState (states.ts:18)
    at Object.isNonTerminalFundingState (states.ts:33)
    at ProtocolContainer.render (container.tsx:26)
    at finishClassComponent (react-dom.development.js:14741)
    at updateClassComponent (react-dom.development.js:14696)
    at beginWork (react-dom.development.js:15644)
    at performUnitOfWork (react-dom.development.js:19312)
    at workLoop (react-dom.development.js:19352)
    at HTMLUnknownElement.callCallback (react-dom.development.js:149)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:199)
    at invokeGuardedCallback (react-dom.development.js:256)
    at replayUnitOfWork (react-dom.development.js:18578)
    at renderRoot (react-dom.development.js:19468)
    at performWorkOnRoot (react-dom.development.js:20342)
    at performWork (react-dom.development.js:20254)
    at performSyncWork (react-dom.development.js:20228)
    at requestWork (react-dom.development.js:20097)
    at scheduleWork (react-dom.development.js:19911)
    at Object.enqueueSetState (react-dom.development.js:11169)
    at Connect.push.../../node_modules/react/cjs/react.development.js.Component.setState (react.development.js:335)
    at Connect.onStateChange (connectAdvanced.js:205)
    at Object.dispatch (redux.js:214)
    at e (<anonymous>:1:40553)
    at middleware.js:66
    at dispatch (redux.js:611)
    at utils.js:250
    at proc.js:489
    at exec (scheduler.js:19)
    at flush (scheduler.js:60)
    at asap (scheduler.js:33)
    at runPutEffect (proc.js:486)
    at runEffect (proc.js:435)
    at next (proc.js:315)
    at Object.currCb [as cont] (proc.js:388)
    at end (proc.js:356)
    at Object.task.cont (proc.js:99)
    at next (proc.js:321)
    at currCb (proc.js:388)
    at runSelectEffect (proc.js:699)
    at runEffect (proc.js:435)
    at next (proc.js:315)
    at proc (proc.js:270)
    at resolveIterator (proc.js:456)
    at runEffect (proc.js:432)
    at next (proc.js:315)
    at currCb (proc.js:388)
    at takeCb (proc.js:466)
    at Object.put (channel.js:73)
    at channel.js:161```

### PR DESCRIPTION
**Pull request by lalexgap**
Updates various reducers to properly use `ProtocolLocator` so funding and concluding actually work.

Fixes #619.

### CODE REVIEW

File: packages/wallet/src/communication/actions.ts [hunk 1]
```diff
@@ -132,7 +132,7 @@ export function isCommonAction(
     (action.type === 'WALLET.COMMON.COMMITMENTS_RECEIVED' ||
       action.type === 'WALLET.COMMON.COMMITMENT_RECEIVED') &&
     // When passed a protocol, check that it's got the protocol in the protocol locator
-    (!protocol || action.protocolLocator.indexOf(protocol) >= 0)
+    (!protocol || (action.protocolLocator && action.protocolLocator.indexOf(protocol) >= 0))
```
Comment (andrewgordstewart) [inline packages/wallet/src/communication/actions.ts]: Why is this needed? Both `COMMITMENTS_RECEIVED` and `COMMITMENT_RECEIVED` have a `protocolLocator`

File: packages/wallet/src/redux/protocols/concluding/responder/reducer.ts [hunk 2]
```diff
@@ -170,6 +170,7 @@ function concludeApproved(protocolState: NonTerminalCState, sharedData: Storage)
       false,
       lastestCommitment.allocation,
       lastestCommitment.destination,
+      makeLocator(CONSENSUS_UPDATE_PROTOCOL_LOCATOR),
```
Comment (andrewgordstewart) [inline packages/wallet/src/redux/protocols/concluding/responder/reducer.ts]: This is counter-intuitive. Is `CONSENSUS_UPDATE_PROTOCOL_LOCATOR` not a protocol locator?

File: packages/wallet/src/communication/actions.ts (hunk 1 above)
Comment (lalexgap) [reply packages/wallet/src/communication/actions.ts]: I should investigate this further, but `isCommonAction` was receiving a `COMMITMENT_RECEIVED` with an `undefined` protocolLocator from a concluding protocol.

Comment (andrewgordstewart) [reply packages/wallet/src/communication/actions.ts]: I thought `COMMITMENT_RECEIVED` now has a mandatory `protocolLocator` property?

File: packages/wallet/src/redux/protocols/concluding/responder/reducer.ts (hunk 2 above)
Comment (lalexgap) [reply packages/wallet/src/redux/protocols/concluding/responder/reducer.ts]: I'll update this. I most likely switched to `makeLocator` before I realized `concluding` is a top level protocol and I thought there would be another `protocolLocator` to pass in.

Comment (andrewgordstewart) [reply packages/wallet/src/redux/protocols/concluding/responder/reducer.ts]: Ah, good point. In that case, the protocolLocator should be `[]`

File: packages/wallet/src/communication/actions.ts (hunk 1 above)
Comment (lalexgap) [reply packages/wallet/src/communication/actions.ts]: This looks like it's due to firebase deciding to strip `protocolLocator:[]` from the action. Going to leave this is as now and we can investigate further alter.
